### ISSUE TITLE
Diagram synthesis fails with NullPointerException for Bank example

### ISSUE THREAD

**Issue by cmnrd** [icyphy/lingua-franca#386]
The following minimal program results in a NullPointerException in the diagram synthesis:
```
target Cpp;
reactor Contained {
    output out:unsigned;
}
main reactor {
    c = new[4] Contained();
    reaction(startup) c.out {==}
}
```
It works fine when removing `[4]` to make `c` a normal reactor instance.

**Comment by edwardalee** [comment]
This is fixed in the `banks-to-reaction` branch.  I have issued a pull request #391.

**Comment by cmnrd** [comment]
closed by #391
